sys/include/xtimer/implementation.h

```diff
@@ -109,24 +107,23 @@ void _xtimer_tsleep(uint32_t offset, uint32_t long_offset);
 #ifndef DOXYGEN
 /* Doxygen warns that these are undocumented, but the documentation can be found in xtimer.h */
 
-static inline uint32_t _xtimer_now(void)
+static inline uint32_t _xtimer_now64(void)
```

Comment:
Comment from afar: The return type and the function name (32 vs 64) seem misleading.

Reply:
Agree. Thanks!